clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -826,8 +858,8 @@ static CompoundStmt *CreateOpenCLKernelBody(Sema &S,
         InitializationSequence InitSeq(S, Entity, InitKind, None);
         ExprResult MemberInit = InitSeq.Perform(S, Entity, InitKind, None);
         InitExprs.push_back(MemberInit.get());
-        getExprForSpecialSYCLObj(FieldType, Field, CRD, KernelObjCloneRef,
-                                 InitMethodName, BodyStmts);
+        getExprForSpecialSYCLObj(FieldType, Field, nullptr, CRD,
```

Comment:
So I see how often you're calling this with nullptr.  I'd prefer that you either move it to the end and make it a default parameter, or make sure that every place you pass nullptr you put the variable-name-comment in it.